app/views/membership_applications/new.html.haml

```diff
@@ -9,3 +9,5 @@
         .wpcf7-response-output
           = msg
     = render partial: 'membership_application_fields', locals: {f: f}
+    %span.req
```

Comment:
Put these 2 lines (12 + 13) into a partial because they're going to be used many places on the site.